docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,40 @@
+| [Kotlin DataFrame](https://github.com/Kotlin/dataframe)                      | - Data collection <br/> - Data cleaning and processing                                                                   | - Operations for creating, sorting, and cleaning data frames, feature engineering, and more  <br/> - Processing of structured data  <br/> - Support for `CSV`, `JSON`, and other input formats <br/> - Reading from SQL databases                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| [Kandy](https://kotlin.github.io/kandy/welcome.html)                         | Data exploration and visualization                                                                                       | - Powerful, readable, and typesafe DSL for plotting charts of various types <br/> - Open-source library written in Kotlin for the JVM <br/> - Support for [Kotlin Notebook](https://kotlin.github.io/kandy/kandy-in-kotlin-notebook.html), [Datalore](https://kotlin.github.io/kandy/kandy-in-datalore.html), and [Jupyter Notebook](https://kotlin.github.io/kandy/kandy-in-jupyter-notebook.html) <br/> - Seamless integration with [Kotlin DataFrame](https://kotlin.github.io/dataframe/gettingstarted.html)                                                                                                                                                 |
+| [KotlinDL](https://github.com/jetbrains/kotlindl)                            | Model building                                                                                                           | - Deep learning API written in Kotlin and inspired by [Keras](https://keras.io/) <br/> - Training deep learning models from scratch or importing existing Keras and ONNX models for inference <br/> - Transfer learning for tailoring existing pre-trained models to your tasks <br/> - Support for [Jupyter Notebook](https://jupyter.org/) and the [Android platform](https://developer.android.com/about)                                                                                                                                                                                                                                                     |    
```

Comment:
Remove please for [Jupyter Notebook](https://jupyter.org/) (stay only with Android)